.scripts/pre-commit.ktlint

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Running KTLint"
+./gradlew ktlintFormat
```

Comment:
Won't this fail as long as `apply plugin: libs.plugins.ktlint.get().pluginId` is commented out?

Reply:
correct but this file has not been symbolically linked to the `.git/hooks/` folder by running the other script `.scripts/install-precommit-git-hook` - engineers would have to run this script manually once to add the ktlint hook to their git

Reply:
I think we probably want that hook installed automatically once we turn this on.